pkg/registry/rbac/clusterrolebinding/policybased/storage.go

```diff
@@ -63,6 +63,10 @@ func (s *Storage) Create(ctx genericapirequest.Context, obj runtime.Object) (run
 	return s.StandardStorage.Create(ctx, obj)
 }
 
+func (s *Storage) CreateInitialized(ctx genericapirequest.Context, obj runtime.Object) (runtime.Object, error) {
+	return s.Create(ctx, obj)
```

Comment:
It is not obvious why it is OK for all of these to just call the create method if they are supposed to be waiting?